utils/install-app.js

```diff
@@ -139,6 +141,78 @@ function addToServer(appDir, installDir) {
   }
 }
 
+function loadRecognizers(appDir, appId, appVers) {
+  let recognizers;
+  let configRecognizers;
+  let configLocation;
+
+  try { // Get recognizers in a plugin's appDir/config/xxx location
+    recognizers = JSON.parse(fs.readFileSync(path.join(appDir, "config/recognizers", appId))).recognizers;
+    const keys = Object.keys(recognizers)
```

Comment:
In the example on your other PR, recognizers is an array, not an object.
{
  "recognizers":[
    {
      "id":"com.rs.iuw.mq.openapp.forward",

Reply:
Yes, it is. It's an array of objects so this is a good way to traverse it. I could also use .map if I wanted to

The rest of searches assume that the elements are 0 through n which I didn't want to do, because it's an unnecessary assumption imo

Reply:
I didn't use Array.keys because it's not supported in IE